zilliqa/src/api/types/eth.rs

```diff
@@ -333,6 +333,14 @@ pub struct TransactionReceipt {
     pub ty: u64,
     #[serde(serialize_with = "bool_as_int")]
     pub status: bool,
+    #[serde(serialize_with = "hex")]
+    pub v: u64,
+    #[serde(serialize_with = "hex")]
+    pub r: U256,
+    #[serde(serialize_with = "hex")]
+    pub s: U256,
+    #[serde(serialize_with = "hex")]
+    pub root: B256,
```

Comment:
Is this field documented anywhere? I can't find references to it in other docs.

Reply:
Its returned by ZQ1, apparently, according to #1756. I assumed it meant the transaction root?

Reply:
I found 

> The post-transaction state root. Only specified for transactions included before the Byzantium upgrade.

on https://docs.metamask.io/wallet/reference/json-rpc-methods/eth_gettransactionreceipt/

This doesn't make sense to return since we aren't following the same forks as Ethereum mainnet and we can assume all Eth transactions are 'post-Byzantium'.

More generally we should be careful to assume that just because ZQ1 returned something, we should do the same. Lots of what ZQ1 does is not sensible and we should avoid copying across that insensibility.

Reply:
Yes - I think these are part of the other Eth specs.

Reply:
Right, I've removed root in the latest push